# CALCULO DE IMC EN PYTHON

In [4]:
# ==========================================
#         CÁLCULO DEL IMC EN PYTHON
# ==========================================

# 1️⃣ Pedir datos al usuario
try:
    # Solicita el peso en kilogramos y lo convierte a float
    peso = float(input("Ingresa tu peso en kg: "))

    # Solicita la altura en metros y lo convierte a float
    altura = float(input("Ingresa tu estatura en metros: "))

    # 2️⃣ Calcular el IMC
    # Fórmula: IMC = peso / (altura^2)
    imc = peso / (altura ** 2)

    # 3️⃣ Mostrar el resultado del IMC
    # round(imc, 2) redondea el valor a 2 decimales
    print(f"\nTu IMC es: {round(imc, 2)}")

    # 4️⃣ Clasificar el IMC según los rangos estándar
    if imc < 18.5:
        print("Clasificación: Bajo peso")
    elif 18.5 <= imc < 25:
        print("Clasificación: Peso normal")
    elif 25 <= imc < 30:
        print("Clasificación: Sobrepeso")
    else:
        print("Clasificación: Obesidad")

except ValueError:
    print("Error: Por favor ingresa números válidos para peso y altura.")



Tu IMC es: 21.13
Clasificación: Peso normal


### EJEMPLO
Poniendo de altura 1.82.<br>
de peso 70 kg.<br>
el resultado es 21.13.<br>
clasficiacion: peso normal.